foam_controlwrapper/foam_controlwrapper.py

```diff
@@ -1,36 +1,37 @@
 """ foam_controlwrapper module
 
-Wrapper module for OpenFOAM control using pyFoam -package
+Wrapper module for OpenFOAM
 
 """
 from simphony.core.cuba import CUBA
 from simphony.core.data_container import DataContainer
 from simphony.cuds.abc_modeling_engine import ABCModelingEngine
-from PyFoam.Execution.ConvergenceRunner import ConvergenceRunner
-from PyFoam.RunDictionary.SolutionDirectory import SolutionDirectory
-from PyFoam.LogAnalysis.BoundingLogAnalyzer import BoundingLogAnalyzer
-from PyFoam.RunDictionary.ParsedParameterFile import ParsedParameterFile
-import os
 from foam_files import FoamFiles
 from cuba_extension import CUBAExt
+from foam_mesh import FoamMesh
+from foam_runner import FoamRunner
+import simphonyfoaminterface
+import os
 
 
 class FoamControlWrapper(ABCModelingEngine):
-    """ Wrapper to OpenFOAM control using pyFoam -package
+    """ Wrapper to OpenFOAM
 
     """
 
     def __init__(self):
 
+        self._meshes = {}
```

Comment:
When subclassing it is good practice to call the base class implementation (especially for constructors). 
In this case it should be:
```python
def __init__(self):
      super(FoamControlWrapper, self).__init__()
      ....
```